docs/content/direct/knownissue-wsl-ghcr-helm.md

```diff
@@ -1,20 +1,62 @@
-# Authorization failure in WSL while fetching Helm chart from ghcr.io
+# Authorization failure in WSL/Ubuntu while fetching Helm chart from ghcr.io
 
 ## Description of the Issue
 
-When using WSL (Windows Subsystem for Linxu) and following the
-[Getting Started recipe](get-started.md) you might get a failure from
-the command to instantiate KubeStellar's core Helm chart. The error
-message is as follows.
+When working with WSL (Windows Subsystem for Linux)/Ubuntu and trying to instantiate KubeStellar's core Helm chart, you might encounter the following error:
 
-> Error: failed to authorize: failed to fetch oauth token: unexpected status from GET request to https://ghcr.io/token?scope=repository%3Akubestellar%2Fkubestellar%2Fcore-chart%3Apull&service=ghcr.io: 403 Fobidden
+> Error: failed to authorize: failed to fetch oauth token: unexpected status from GET request to https://ghcr.io/token?scope=repository%3Akubestellar%2Fkubestellar%2Fcore-chart%3Apull&service=ghcr.io: 403 Forbidden
 
-This is [Issue 2544](https://github.com/kubestellar/kubestellar/issues/2544).
+This error indicates that there was an issue fetching the OAuth token from GitHub's container registry (`ghcr.io`), which is preventing you from pulling the Helm chart.
```

Comment:
I would add a remark about diagnosing this problem with the simpler command `helm show chart oci://ghcr.io/kubestellar/kubestellar/core-chart`.

Reply:
Another diagnostic would be to test whether https://github.com/actions/actions-runner-controller/issues/3381#issuecomment-2619817888 solves the problem.